From 31b48b2dfcb64c1f4fa4db2940cecc557b908500 Mon Sep 17 00:00:00 2001
From: Simon McVittie <smcv@debian.org>
Date: Sat, 21 May 2016 15:28:17 +0100
Subject: [PATCH 4/7] Increase Priority to optional

This tool is likely to be depended on by gnome-software (via Flatpak) in
future.
---
 debian/changelog | 2 ++
 debian/control   | 2 +-
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/debian/changelog b/debian/changelog
index 2d464a3..e85d381 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -4,6 +4,8 @@ bubblewrap (0~git160513-3) UNRELEASED; urgency=medium
   * debian/control: make the whole package Linux-only. Like Flatpak, this
     package is inherently non-portable.
   * Move from Section: web to Section: admin
+  * Increase Priority to optional, because this tool is likely to be
+    depended on by gnome-software (via Flatpak) in future
 
  -- Simon McVittie <smcv@debian.org>  Sat, 21 May 2016 15:10:56 +0100
 
diff --git a/debian/control b/debian/control
index e0ad3d5..e43e10e 100644
--- a/debian/control
+++ b/debian/control
@@ -1,6 +1,6 @@
 Source: bubblewrap
 Section: admin
-Priority: extra
+Priority: optional
 Maintainer: Laszlo Boszormenyi (GCS) <gcs@debian.org>
 Build-Depends: debhelper (>= 9), dh-autoreconf, pkg-config, libselinux1-dev (>= 2.1.9), libcap-dev, bash-completion, xsltproc, docbook-xsl
 Standards-Version: 3.9.8
-- 
2.8.1

